### ISSUE TITLE
add after kill command

### ISSUE THREAD

**Issue by phra** [kimmobrunfeldt/concurrently#103]
hi,

it would be nice to have to ability to invoke a sort of fallback command when the `kill-other` condition is reached, e.g. it can be a command to stop the test database if the lint task is failing when launching tests in parallel with the linting task.

**Comment by gustavohenke** [comment]
Perhaps like this?

```
concurrently --kill-others-on-fail "echo foo" "echo bar" || echo "someone died"
```

**Comment by phra** [comment]
@gustavohenke yeah, but my goal is to remain agnostic about the shell, e.g. the `||` bash operator is not available in fish, so that `echo someone died` should be launched from concurrently itself to avoid using shell-specific operators.

**Comment by gustavohenke** [comment]
Okay. If you want to try to tackle this, please go ahead!

**Comment by phra** [comment]
if i have spare time i will do it! @gustavohenke

**Comment by filipesilva** [comment]
An implementation of https://github.com/kimmobrunfeldt/concurrently/issues/101 would allow for this usage pattern (and others).

**Comment by gustavohenke** [comment]
👆 programmatic API exists for a while. Closing.
